getting-started.md

```diff
@@ -41,16 +41,20 @@ Robolectric works with [Bazel](https://bazel.build) 0.10.0 or higher. Bazel inte
 
 Robolectric needs to be added as a dependency to your Bazel project. Add the following to your WORKSPACE file:
 ```python
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
 http_archive(
  name = "robolectric",
- urls = ["https://github.com/robolectric/robolectric/archive/<COMMIT>.tar.gz"],
- strip_prefix = "robolectric-<COMMIT>",
+ urls = ["https://github.com/robolectric/robolectric-bazel/archive/robolectric-bazel-<VERSION>.tar.gz"],
```

Comment:
I'd consider having the docs point to the most updated version so that this is copy-pastable and users don't need to guess and then you can add the sha256 value too